packages/host/app/commands/create-ai-assistant-room.ts

```diff
@@ -0,0 +1,60 @@
+import { service } from '@ember/service';
+
+import format from 'date-fns/format';
+
+import { aiBotUsername } from '@cardstack/runtime-common';
+
+import type * as BaseCommandModule from 'https://cardstack.com/base/command';
+
+import HostBaseCommand from '../lib/host-base-command';
+
+import { AI_BOT_POWER_LEVEL } from '../services/matrix-service';
```

Comment:
maybe MatrixService should just expose this as a property, avoiding the implementation dep on matrix-service and the concordant module scope eval of matrix service